changes/README.md

```diff
@@ -10,16 +10,17 @@ A news file should be added to this directory for each PR.
 
 On release of the action, the content of the file becomes part of the [change log](../CHANGELOG.md) and this directory is reset.
 
-## News Files
+### News Files
```

Comment:
This duplicate the contribution guide, can some of this be removed and link to the guide put in it's place?

Reply:
Done